CHANGELOG.md

```diff
@@ -14,17 +14,30 @@
 
 - Update DSE to 6.8.25 [\#2022](https://github.com/stargate/stargate/issues/2022)
 - CQL module needs to be upgraded [\#2015](https://github.com/stargate/stargate/issues/2015)
+- Avoid creating a new grpc stub for each request in the `StargateRequestInfo` [\#2004](https://github.com/stargate/stargate/issues/2004)
+- `StargateV1ConfigurationSourceProviderTest` failing on v2 branch [\#2001](https://github.com/stargate/stargate/issues/2001)
+- Move all generic and shared components to the `svg2-quarkus-common` [\#1983](https://github.com/stargate/stargate/issues/1983)
```

Comment:
@tatu-at-datastax Wait how does it work with issues related to V2? They are part of the closed issues?

Reply:
They should not be included in V1 release notes, so manually remove.

Reply:
Done in https://github.com/stargate/stargate/pull/2033/commits/b5d7dbeceee0d75fd489ffa0e58c3a82082c6e80